### DISCUSSION TITLE
Why is my name always a keyword?

### DISCUSSION THREAD

**Discussion by SwedenGuy** [photoprism/photoprism#4195] [Q&A]
In every photo that I have indexed, my name (Josh) is always one of the system generated labels.  Doesn't matter if I am in the photo or not. Is this being picked up from the import file name somehow?  

Eg: Filename: Josh’s iPhone 12/2024-04-12_14-54-27.MOV.jpg
Keywords: ejdervägen, green, josh, klosters-fälad, lund, papegojelyckan, sweden

**Comment by lastzero** [comment]
Yes. It's a keyword since it is found in the file path.

**Reply by SwedenGuy** [reply]
Appreciate the quick answer. But that is really strange logic.  What does file path have to do with the contents of the photo (especially when being imported from a phone)?

**Reply by lastzero** [reply]
With the file path `2002/NYC 2002/P3110049.JPG`, for example, you can find all pictures contained in this folder by searching for "nyc". PhotoPrism will then also suggest that the pictures were taken in the United States (if no other information is available):

![Screenshot 2024-04-19 at 17 22 39](https://github.com/photoprism/photoprism/assets/301686/87562dd6-ad12-4fc7-90f8-3a0b958e4554)

**Note that we also have a stop word list for terms which should be ignored when extracting keywords from file and folder names (such as img, iphone, image, photo,....), but of course we don't have every user's personal name on it.**

**Reply by SwedenGuy** [reply]
I can understand the general logic if importing from a memory or usb card, but seems a bit overkill if the import is from a phone using Photosync, as there is no way to change the name short of changing the name of the phone, Is there any way to access the stop word list?  Eg, a config file somewhere?

**Reply by lastzero** [reply]
PhotoSync lets you configure the target folder and filenames:
![IMG_4935](https://github.com/photoprism/photoprism/assets/301686/d79f0ac6-bad6-4148-806c-9d11f666db60)
